.eslintrc.json

```diff
@@ -43,7 +43,7 @@
         "@typescript-eslint/no-this-alias": "warn",
         "no-dupe-class-members": "warn",
         "no-prototype-builtins": "warn",
-        "no-unused-vars": "warn",
+        "no-unused-vars": "off",
```

Comment:
This issue show up twice in the log as it is picked up by ESLint and TypeScript ESLint

```bash
npm run lint
<snip>
src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.component.ts        
   14:26   warning  'Unit' is defined but never used                no-unused-vars                                                               
   14:26   warning  'Unit' is defined but never used                @typescript-eslint/no-unused-vars     
<snip>
```